rswag-specs/spec/rswag/specs/request_factory_spec.rb

```diff
@@ -151,6 +151,28 @@ module Specs
           end
         end
 
+        context "'query' parameter of format 'datetime'" do
+          let(:date_time) { DateTime.now.to_s }
```

Comment:
minor quibble: Imo I'd be nice to have a static `DateTime` value so that the tests could look for a specific output rather than the output of a specific function (aka. testing the result, not the means of achieving it)
eg.

```rb
let(:date_time) { DateTime.new(2001, 2, 3, 4, 5, 6, '-7').to_s }  # "2001-02-03T04:05:06-07:00"

 # ...
 
expect(request[:path]).to eq('/blogs?date_time=2001-02-03T04%3A05%3A06-07%3A00')
```